### ISSUE TITLE
Selective sharing between multiple users?

### ISSUE THREAD

**Issue by i-schuetz** [realm/realm-cocoa#4379]
Hi,

I read the sync thread, which introduces the realm platform at the end: https://github.com/realm/realm-cocoa/issues/913

I haven't been able to find if it's possible to add multi-user features, like (real time) sharing of specific items (with invitations, etc). Is there a way to do this? Is it at least possible to use this platform as base or in parallel which such features?

Thanks!

**Comment by jpsim** [comment]
There's an existing conversation happening in our "meta" issue tracker about the Realm Mobile Platform as a whole: realm/realm-mobile-platform#52

You can also see past work (#4324) and work in progress (#4365) related to this if you're curious in reading up on this, but rest assured that we'll announce and document features around this as they're ready.

**Comment by i-schuetz** [comment]
Thanks! It's great to see there's work in progress here.

What I need may be a bit too fine-grained, at least for the current state. I'd want to share single records. For example a user has lists. They can decide to share some of these lists with other people. This means only these records + list items that are associated with these lists would be shared. Is something like this included in the current developments/roadmap? The linked discussion is a little too general and the pull requests on the other side too specific/internal to understand if this is the case.

**Comment by jpsim** [comment]
Yeah, we have things planned for exactly that, but since we have some releases and blog posts announcing and explaining that in more detail scheduled to come out in the coming weeks and months, I'll ask you to wait until those are out for more information. In the meantime, I'll close this ticket as a duplicate of realm/realm-mobile-platform#52 and ask you to please subscribe to that ticket and continue the conversation there.
